Source/MultipartFormData.swift

```diff
@@ -256,16 +256,17 @@ public class MultipartFormData {
 
         var isReachable = false
 
-        do {
-            try fileURL.checkResourceIsReachable()
-            isReachable = true
-        } catch {
-            // No-op
+        if #available(OSX 10.10, *) {
```

Comment:
I don't think so. The deployment target is set to 8.0, which is the OS version where this API was introduced.